client/packages/system/src/Patient/Insurance/InsuranceProvidersSelect.tsx

```diff
@@ -0,0 +1,53 @@
+import React, { FC, ReactElement } from 'react';
+
+import { useTranslation } from '@common/intl';
+import { Autocomplete, InputWithLabelRow } from '@common/components';
+
+import { useInsuranceProviders } from '../apiModern/hooks';
+
+interface InsuranceProvidersSelectProps {
+  insuranceProviderId: string;
+  onChange: (value: string) => void;
+}
+
+export const InsuranceProvidersSelect: FC<InsuranceProvidersSelectProps> = ({
+  insuranceProviderId,
+  onChange,
+}): ReactElement => {
+  const t = useTranslation();
+  const { data } = useInsuranceProviders();
+  const insuranceProviders = data?.nodes ?? [];
```

Comment:
Minor thing, and I don't think we've been particularly consistent with it, but I think it's nicer to return `data.nodes ?? [ ] ` from the hook, so consumers just get the "data" directly without having to check and get the `nodes` array.

(For example `usePrescriptionList`)

Reply:
Great idea! Applied here: https://github.com/msupply-foundation/open-msupply/pull/6546/commits/1fa87f5b747d724e7928c9b81ef3dd4f57207966